### PR TITLE
File path access changes

### PR DESCRIPTION
**Pull request by juhlee-microsoft**
Add to relative path to, so that it can be called in any place.

### CODE REVIEW

**Comment by juhlee-microsoft** [comment]
Test Results
------------------------------------------------------
PS C:\Users\juhlee\Source\Repos\LISAv2> cd ..
PS C:\Users\juhlee\Source\Repos> .\LISAv2\Utilities\Get-LISAv2Statistics.ps1

TestCase                                                                 Platform        Category            Area                                     Tags
----------------------------------------------------------------------------------------------------------------------------------------------------------
BVT-VERIFY-DEPLOYMENT-PROVISION                                      Azure,HyperV             BVT             BVT                                 bvt,boot
BVT-IS-ROOT-PASSWORD-DELETED                                                Azure             BVT             BVT                                      bvt
BVT-ROOT-DEVICE-TIMEOUT-CHECK                                               Azure             BVT             BVT                                      bvt

File: Utilities/Get-LISAv2Statistics.ps1 [hunk 1]
```diff
@@ -4,7 +4,9 @@
 # Description: This script displays the LISav2 test case statistics and list of available tags.
 
 # Read all test case xml files
-$files = Get-ChildItem ..\XML\TestCases\*.xml -Exclude Other.xml
+$_currentPath = Split-Path -Parent $MyInvocation.MyCommand.Path
```
Comment (lizzha) [inline Utilities/Get-LISAv2Statistics.ps1]: why the same starts with underscore?

Comment (juhlee-microsoft) [reply Utilities/Get-LISAv2Statistics.ps1]: @lizzha This is just a temp variable. Do you want me to remove underscore?

Comment (lizzha) [reply Utilities/Get-LISAv2Statistics.ps1]: yes please, since we don't use this naming convention in LISAv2

Comment (juhlee-microsoft) [reply Utilities/Get-LISAv2Statistics.ps1]: fixed d34ce5ca55cd9881834103732ab5ed22d5ff3bfd

**Pull request merged**